.gitignore

```diff
@@ -1,3 +1,9 @@
 .DS_Store
 scripts/test.sh
 build/*
+node_modules
+.vscode
```

Comment:
So, `node_modules` and `.vscode` don't belong in the tree at all. An `npm install` (part of the make files...) will populate `node_modules` and `.vscode` is their crazy launch config for debugging (actually pretty cool I can debug 9volt).